internal/components/exporters_test.go

```diff
@@ -125,6 +126,14 @@ func TestDefaultExporters(t *testing.T) {
 				return cfg
 			},
 		},
+		{
+			exporter: "parquet",
+			getConfigFn: func() config.Exporter {
+				cfg := expFactories["parquet"].CreateDefaultConfig().(*parquetexporter.Config)
+				cfg.Path, _ = ioutil.TempDir("", "*")
```

Comment:
Not sure I would want a temp dir being made each time the test is run without being cleaned up.

Would `/tmp` be enough here?

Reply:
`/tmp` might not be accurate depending on the platform, TempDir is indeed correct. The test does need to clean up after itself though.

Reply:
Makes sense to me.

Reply:
Moved the folder to be cleaned up.